src/main/java/org/jahia/modules/sitemap/services/SitemapService.java

```diff
@@ -23,24 +23,29 @@
  */
 package org.jahia.modules.sitemap.services;
 
+import net.sf.ehcache.Ehcache;
 import org.jahia.modules.sitemap.exceptions.SitemapException;
 
-import javax.jcr.RepositoryException;
-
 public interface SitemapService {
 
+    Boolean sendSitemapXMLUrlPathToSearchEngines(String sitemapIndexXMLUrlPath) throws SitemapException;
+
     /**
-     * TODO add java doc
-     * @param sitemapIndexXMLUrlPath
-     * @return
-     * @throws SitemapException
+     * Gets sitemap ehcache.
+     * @return sitemap Ehcache.
      */
-    Boolean sendSitemapXMLUrlPathToSearchEngines(String sitemapIndexXMLUrlPath) throws SitemapException;
+    Ehcache getSitemapEhCache();
```

Comment:
the service should not expose the cache, but methods to manipulate the cache (add / flush / get)